scripts/local-docker.env

```diff
@@ -6,6 +6,7 @@ export MARLOWE_CHAINSEEKD_COMMAND_PORT=$(docker-compose port chainseekd 3720 | s
 export MARLOWE_RT_SYNC_HOST=127.0.0.1
 export MARLOWE_RT_SYNC_MARLOWE_SYNC_PORT=$(docker-compose port marlowe-sync 3724 | sed -e s/://)
 export MARLOWE_RT_SYNC_MARLOWE_HEADER_SYNC_PORT=$(docker-compose port marlowe-sync 3725 | sed -e s/://)
+export MARLOWE_RT_SYNC_MARLOWE_QUERY_SYNC_PORT=$(docker-compose port marlowe-sync 3726 | sed -e s/://)
```

Comment:
Thanks! This reminds me we need to add `3726` to the `ports` exposed by `sync-service` in `compose.nix`